automatic/selenium-chromium-edge-driver/update.ps1

```diff
@@ -36,14 +34,14 @@ function global:au_SearchReplace {
 }
 
 function global:au_GetLatest {
-  # We use the available package on CCR as the source of truth
-  $packageVersion = Get-EdgePackageVersion -PackageName 'microsoft-edge'
-  $escapedVersion = [regex]::Escape($packageVersion)
+  $edgeDriverStableVersionURL = "https://msedgewebdriverstorage.blob.core.windows.net/edgewebdriver/LATEST_STABLE"
+  $stableVersionFile = "$env:TEMP\EdgeDriverLatestStable.txt"
 
-  $downloadPage = Invoke-WebRequest $releases -UseBasicParsing
+  Invoke-WebRequest -URI $edgeDriverStableVersionURL -OutFile $stableVersionFile -UseBasicParsing
+  $packageVersion = Get-Content $stableVersionFile
 
-  $url32 = $downloadPage.Links | Where-Object { $_.href -match "$escapedVersion.*win32" } | Select-Object -First 1 -ExpandProperty href
-  $url64 = $downloadPage.Links | Where-Object { $_.href -match "$escapedVersion.*win64" } | Select-Object -First 1 -ExpandProperty href
+  $url32 = "https://msedgedriver.azureedge.net/$packageVersion/edgedriver_win32.zip"
+  $url64 = "https://msedgedriver.azureedge.net/$packageVersion/edgedriver_win64.zip"
 
   if (!$url32 -or !$url64) {
```

Comment:
The code changes above makes this redundant as `$url32` and `$url64` will not be empty.

Reply:
Correct. Thx for the pointer